dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -369,6 +365,19 @@ static StaticLinkageChecker getInstanceFromArguments(String[] arguments) throws
     return pathToClasses;
   }
 
+  private static ImmutableSet<JavaClass> topLevelJavaClassesInJar(Path jarFilePath,
+      SyntheticRepository repository) throws IOException, ClassNotFoundException {
+    Set<JavaClass> javaClasses = new HashSet<>();
```

Comment:
If you'd like, you can use `ImmutableSet.Builder<JavaClass> javaClasses = ImmutableSet.builder();`. The builders have `add` methods, and then you just call `build()` to get the set.

Reply:
That's nice. Now the code uses `ImmutableSet.Builder`